datarequest.py

```diff
@@ -236,6 +238,26 @@ def transform_title(row):
                         ('items', colls)])
 
 
+@api.make()
+def api_schema_get(ctx, schema_name):
+    """Get a schema.
+
+       Arguments:
+       schema_name -- Name of schema.
+    """
+    # Define paths to schema and uischema
+    coll_path = "/" + user.zone(ctx) + "/" + SCHEMACOLLECTION
```

Comment:
Use Python format for readability: https://docs.python.org/3/library/stdtypes.html#str.format

Also prepares for easier transition to Python 3 f strings: https://www.python.org/dev/peps/pep-0498/